src/SwedbankPay/Api/Client/Client.php

```diff
@@ -62,8 +62,8 @@ public function __construct($data = [])
 
         // Set version
         if (!$this->offsetExists(self::CLIENT_VERSION)) {
-            $data = json_decode(file_get_contents(__DIR__ . '/../../../../composer.json'), true);
-            $this->setVersion($data['version']);
+            $version = Version::getVersion();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid using static access to class 'SwedbankPay\Api\Client\Version' in method '__construct'.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-php/pullRequest?prid=5251412)